### PR TITLE
[LasercutterTechdrawExport] exports one contour for each solid in each selected object

### PR DESCRIPTION
**Pull request by chbergmann**
This macro is also present as a tool in the CurvedShapes workbench

Thank you for creating a pull request to contribute to FreeCAD-macros!
To integrate your macro please make sure the following steps are complete:

- [ ] Please check this box if you're not submitting a new macro.   
- [ ] Are you submitting a new macro ?  
- [ ] Have you followed the ['How to submit a macro'](../README.md#how-to-submit-a-macro) section of the README.md ?  
- [ ] Your macro has a [Description](../README.md#macro-description) in its header.  
- [ ] Your macro has a [CamelCase name](../README.md#camelcase-macro-name).  
- [ ] Your macro is named [appropriately](../README.md#macro-name-specifics).  
- [ ] Your macro contains a [Metadata section](../README.md#macro-metadata) that immediately follows the header description.  
- [ ] Your macro is Python3/Qt5 compliant and tested on the latest FreeCAD stable and development releases.  
- [ ] You're including documentation on how your macro works (bonus: screenshots and/or video on the Wiki)  
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)  
- [ ] Commit message is titled in the following way `[MacroName] Short description`.
- [ ] Optional, write or update the changelog in the macro, from latest to oldest.

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: TechDraw/LasercutterSVGExport.FCMacro [hunk 1]
```diff
@@ -21,167 +21,304 @@ __Files__ = 'LasercutterSVGExport/Readme.md,LasercutterSVGExport/LasercutterSVGE
...
+        # Generate commands in the FreeCAD python console to create LasercutterTechdrawExport
+        FreeCADGui.doCommand("from LasercutterTechdrawExport import LasercutterTechdrawExport")
+        
+        FreeCADGui.doCommand("parts = []")
+        selection = FreeCADGui.Selection.getSelectionEx()
+        for sel in selection:
+            FreeCADGui.doCommand("parts.append(FreeCAD.ActiveDocument.getObject('%s'))"%(sel.ObjectName))
+            
+        FreeCADGui.doCommand("LasercutterTechdrawExport.makeLasercutterTechdrawExport(parts, BeamWidth = %f, doc = App.activeDocument())"%self.form.doubleSpinBox.value())
+        self.form.close()
```
Comment (galou) [inline TechDraw/LasercutterSVGExport.FCMacro:229]: Add `FreeCADGui.doCommand('del __parts').

**Review by galou — CHANGES_REQUESTED** [review]
Thanks for you contribution. I'll merge after the change from `FreeCADGui` to `gui`.

**Comment by luzpaz** [comment]
ping @chbergmann

**Comment by chbergmann** [comment]
I think I fixed everything now. Thank you for the review.

**Comment by galou** [comment]
Thanks!

**Pull request merged**